services/workflows-service/scripts/alerts/generate-alerts.ts

```diff
@@ -133,7 +137,39 @@ export const ALERT_INLINE_RULES = [
       } as TransactionsAgainstDynamicRulesType,
     },
   },
-] as const satisfies readonly { inlineRule: InlineRule; defaultSeverity: AlertSeverity }[];
+] as const satisfies readonly {
+  inlineRule: InlineRule;
+  label: string;
+  defaultSeverity: AlertSeverity;
+}[];
+
+function createData(
```

Comment:
Does ESLint not error on this function in the IDE?

Reply:
No error on my machine. But I'll change it to an arrow function :)